cmd/nvidia-validator/main.go

```diff
@@ -649,7 +652,7 @@ func setEnvVar(envvars []string, key, value string) []string {
 // For driver container installs, check existence of .driver-ctr-ready to confirm running driver
 // container has completed and is in Ready state.
 func assertDriverContainerReady(silent bool) error {
-	command := "bash"
+	command := shell
```

Comment:
Out of scope for this PR, but it may be cleaner and easy to reason about if we implement a "Runner" here (see something like https://github.com/NVIDIA/nvidia-container-toolkit/blob/81fb7bb9c117d653a94d41e6a8bad7019537db9e/tests/e2e/runner.go#L82-L102) where we could better encapsulate the shell specifics.